JSDemos/Demos/List/ItemTemplate/description.md

```diff
@@ -1 +1,3 @@
-This demo shows how you can apply a&nbsp;custom template to&nbsp;items in&nbsp;the **List** UI component.
\ No newline at end of file
+The **List** supports item customization. You can use predefined customization settings. To do so, define [specific fields](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) in item data object.
```

Comment:
I think we should elaborate on default templates (what you call "predefined customization settings") a bit more.

```suggestion
To customize list items, you can use a default item template or implement a custom template.

A default template is a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use the default template for customization, simply add or remove the fields to or from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.
```